### ISSUE TITLE
Connect a hosted TCP service 

### ISSUE THREAD

**Issue by Abhijeet310187** [hikalkan/scs#33]
Hi,

We have a logging service hosted in our infrastructure, that is built using HIK SCS library.

Is it possible to connect to the service normally without HIK SCS client?

I am getting issues.

**Comment by hikalkan** [comment]
> Is it possible to connect to the service normally without HIK SCS client?

Yes, possible if you can implement the same protocol on the pairing application. You can understand protocol (used [BinarySerializationProtocol ](https://github.com/hikalkan/scs/blob/master/src/Scs/Communication/Scs/Communication/Protocols/BinarySerialization/BinarySerializationProtocol.cs) by default) from the document (https://www.codeproject.com/Articles/155282/A-Complete-TCP-Server-Client-Communication-and-RMI) and source code.

Hint: See to understand the message format of the BinarySerializationProtocol class:

https://github.com/hikalkan/scs/blob/master/src/Scs/Communication/Scs/Communication/Protocols/BinarySerialization/BinarySerializationProtocol.cs#L60

There is a 4-bytes "message length" followed by the binary serialized object (for each message).

**Comment by Abhijeet310187** [comment]
Thank you Halil,

I will give it  a try.

Regards,
Abhijeet
